calculator/operations.py

```diff
@@ -1,23 +1,49 @@
 
 def add(*args):
-    # your implementation here
-    pass
+    output = 0
+    count = 0
+    for arg in args:
```

Comment:
You aren't doing anything with `count` so it doesn't need to be here. Also you don't need the if statements, what you have in your `else` will handle the first run through the loop just fine. (It would just be adding to zero)